app/scripts/controllers/transactions/pending-tx-tracker.js

```diff
@@ -211,7 +211,7 @@ export default class PendingTransactionTracker extends EventEmitter {
     const { hash: txHash, txParams: { nonce, from } } = txMeta
     const networkNextNonce = await this.query.getTransactionCount(from)
 
-    if (parseInt(nonce, 16) >= parseInt(networkNextNonce, 16)) {
+    if (parseInt(nonce) >= parseInt(networkNextNonce)) {
```

Comment:
[To quote the MDN article for `parseInt`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt):
>**always specify a _`radix`_ when using `parseInt`.**

We should continue to use a radix for the `nonce` since it is a hex string.

Perhaps we could do something like this instead?

```suggestion
    if (parseInt(nonce) >= networkNextNonce.toNumber()) {
```